### DISCUSSION TITLE
PatchTool 6.57 3/6

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#422] [Announcements]
更新漢化至6.50。

請從release下方連結或內文連結下載檔案。以下兩種方法擇一使用即可。

## 方法一：直接覆蓋遊戲檔案

解壓縮後直接覆蓋`SquareEnix\FINAL FANTASY XIV - A Realm Reborn\game\sqpack\ffxiv`內的同名檔案即可。內含三個000000開頭的字體相關檔案與三個0a0000開頭的文本相關檔案。

設定：Windows作業系統，非高清字體，以繁體中文覆蓋日文和英文，官方渠道版本。

設定相同的人可以考慮直接下載這包，（可能）比較不會遇到問題。其他作業系統未測試。

- ffxiv-20240306.rar：請從下方連結下載。

## 方法二：使用漢化器

使用方法請參照[readme](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/2.-%E4%BD%BF%E7%94%A8)。

檔案：

- FFXIVChnTextPatch-20240306.rar：漢化器本體。需要自行調整設定的人請下載這包手動漢化。請從下方連結下載。

視需求選用：

- [font-custom-4k-smaller-20230131.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.0.1/font-custom-4k-smaller-20230131.rar)：高畫質字體包，請取代原本的resource/font資料夾。[使用此字體包需更改設定。](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)透過重複使用字體字型縮小檔案體積，使用Noto Sans。
- [font-custom-20230111.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.72.2/font-custom-20230111.rar)：舊高畫質字體包，請取代原本的resource/font資料夾。[使用此字體包需更改設定。](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)所有大小字體皆分開繪製，使用Noto Sans和Noto Serif。
- rawexd-zhcn-20240306.rar：簡體中文CSV，取代原本resource/rawexd資料夾。
- [FFXIVChnTextPatch-20230603.jar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.1.1.0/FFXIVChnTextPatch-20230603.jar)：如果你有遇到JNI error，請參考 https://github.com/GpointChen/FFXIVChnTextPatch-GP/issues/90 放到與exe同位置使用。
- [backup-20240214.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.2.1.3/backup-20240214.rar)：乾淨的0X0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。只適用於這個release發布時的遊戲版本。

※若不替換字體會導致遊戲內無法正常顯示部分中文字。另外，目前大廳部分中文字必須使用高畫質字體包才能正常顯示。更多資訊可參閱[wiki頁面](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v3.2.1.4'>PatchTool 6.57 3/6</a>.</em>

**Comment by ETplanet** [comment]
過往漢化程式路徑可以修改成功
但是0306這個檔案我修改到我自己的路徑他就會跳出錯誤不讓我選擇路徑
不太確定哪邊出了錯誤
我的路徑比預設的多了一個"FF14"
但過往的漢化程式我都可以選擇我自己這個路徑沒問題
官方建議的>D:\FFXIV\SquareEnix\FINAL FANTASY XIV - A Realm Reborn
我的>D:\FF14\SquareEnix\FINAL FANTASY XIV - A Realm Reborn
錯誤如圖
![image](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/164009011/53d661cc-6aeb-4b84-9ba2-31577400a935)

**Reply by ETplanet** [reply]
抱歉更新回報
貌似不是0306檔案問題
今天的FF14更新後過往舊的漢化程式也會在修改路徑時跳出錯誤
